app/views/miq_ae_class/_method_inputs.html.haml

```diff
@@ -50,6 +56,38 @@
           :read_only               => true}
     - elsif @ae_method.location == 'expression'
       = @expression
+    - elsif @ae_method.location == 'playbook'
+      .form-horizontal.static
+        .form-group
+          %label.col-md-2.control-label
+            = _('Repository')
+          .col-md-8
+            = h(@playbook_details[:repository])
```

Comment:
we should have a helper for the block
```
form-group
￼ +          %label.col-md-2.control-label
￼ +            = _('Repository')
￼ +          .col-md-8
￼ +            = h(@playbook_details[:repository])
```
`= format_form_group(_('Repository'), @playbook_details[:repository])`